server/src/universal.ts

```
@@ -3,18 +3,23 @@ import {
   minifySelfGameData,
   findRoomWithConnectionID,
   checkIfPropertyWithValueExists,
-  getRank
+  getRank,
+  generateGuestID,
+  generateConnectionID
 } from "./core/utilities";
 import { log } from "./core/log";
-
+import { validateCustomGameSettings } from "./core/utilities";
+import fs from "fs";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)